### ISSUE TITLE
Support for Accessibility is missing

### ISSUE THREAD

**Issue by jagadeshpaladugula** [ashishgkwd/ngx-mat-daterange-picker#22]
**Describe the bug**
Not able to open the date range picker using key board

**To Reproduce**

**Expected behavior**
Date range picker should pass all accessibility rules.

**Screenshots**

**Desktop (please complete the following information):**
 - Browser [e.g. chrome, safari] - Tried with Safari

**Smartphone (please complete the following information):**

**Additional context**
